src/lib/Chip/types.ts

```diff
@@ -0,0 +1,8 @@
+export enum Variants {
```

Comment:
I would put this `enum` directly into the `Chip` component, inside a `<script context="module">`.

Reply:
Instead of being an `enum`, the `Variants` type could be defined as

```suggestion
export type Variants = 'default' | 'info' |  'success' | 'warning' | 'danger';
```

the reason is the `enum` is not being used such, it is just to get its keys. I believe this would simplify and make explicit its usage.

Reply:
If we move this type into `Chip.svelte`, we can move the component to the `lib` folder.